testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -204,9 +220,18 @@ def test_set_residues(self):
 
 
 class TestResnames(TestResidueAttr):
-    values = np.array(['ARG', 'LYS', 'VAL', 'POPG'], dtype=np.object)
+    values = np.array(['VAL', 'LYS', 'VAL', 'POPG'], dtype=np.object)
     attrclass = tpattrs.Resnames
 
+    def test_get_named_residue(self):
```

Comment:
How/where can I trigger that code-path with normal atomgroup objects? For a Atomgroup I can see that it's called in `getattr__` which is transplanted to `__getattr__`. This is also for some others the case but not for resnames. I could just call these methods manually but I would like to call them from the highlevel interface to check that it triggers the right exceptions at the right place in the code.